README.md

```diff
@@ -271,6 +286,14 @@ Either
 
 You can find installation instruction for Gradle on the [Gradle website](https://gradle.org/install/).
 
+### Minimum supported Gradle version is x.x.x. Current version is x.x.x. If using the gradle wrapper, try editing the distributionUrl
+
+Make sure you are have `CORDOVA_ANDROID_GRADLE_DISTRIBUTION_URL` in your path or export it to particular terminal session with correct distribution enabled eg:
+
+>CORDOVA_ANDROID_GRADLE_DISTRIBUTION_URL=https://services.gradle.org/distributions/gradle-x.x.x-all.zip
+
+Bear in mind that other Cordova projects you have might also effected by this change.

```

Comment:
Bear in mind that other Cordova projects would also be affected by this change.

Reply:
Done